packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -0,0 +1,297 @@
+  }
+
+  private calculatePaddingX(): string {
+    if (this.horizontal) return '';
+    if (this.relaxed && this.indented) return 'pl-8 pr-4';
+    if (this.relaxed) return 'px-4';
+    if (this.indented) return 'pl-4';
+    return '';
+  }
+
+  render() {
+    const slots = {
+      label: this.hasSlotController.test('[default]'),
+      'icon-left': this.hasSlotController.test('icon-left'),
+      'icon-right': this.hasSlotController.test('icon-right'),
+      main: this.hasSlotController.test('main'),
+      description: this.hasSlotController.test('description'),
+      children: this.hasSlotController.test('children')
+    };
+    const tag = this.isLink() ? literal`a` : slots['children'] ? literal`summary` : literal`button`;
+
+    // styles
+    const horizontalPaddingBottom = this.horizontal ? 'pb-2' : 'pb-3';
+    const hostDisplayStyle = this.horizontal
+      ? html`<style>
+          :host {
+            display: inline-block;
+          }
+        </style>`
+      : html`<style>
+          :host {
+            display: block;
+          }
+        </style>`;
+
+    // conditional elements
+    const chevron =
+      (this.chevron || slots['children']) && !this.horizontal
+        ? html`<sd-icon
+            name="chevron-down"
+            part="chevron"
+            library="system"
+            color="currentColor"
+            class=${cx('h-6 w-6', this.isAccordion() ? (this.open ? 'rotate-180' : 'rotate-0') : 'rotate-[270deg]')}
+          ></sd-icon>`
+        : null;
+
+    const divider =
+      this.divider && !this.horizontal
+        ? html`<sd-divider
+            part="divider"
+            class=${cx(
+              'w-[calc(100%-64px)] absolute top-[-1px] left-0',
+              this.horizontal ? 'mx-4' : 'mx-8',
+              this.calculatePaddingX()
+            )}
+          ></sd-divider>`
+        : null;
+
+    const iconLeftSlot = slots['icon-left']
+      ? html`<slot name="icon-left" part="icon-left" class="inline-flex justify-center items-center mr-2"></slot>`
+      : null;
+
+    const iconRightSlot = slots['icon-right']
+      ? html`<slot name="icon-right" part="icon-right" class="inline-flex justify-center items-center mr-2"></slot>`
+      : null;
+
+    const mainSlot =
+      slots['main'] && !this.horizontal
+        ? html`<slot name="main" part="main" class=${cx('inline-flex justify-center items-center mr-4')}></slot>`
+        : null;
+
+    const descriptionSlot =
+      slots['description'] && !this.horizontal
+        ? html`<slot
+            name="description"
+            part="description"
+            class=${cx(
+              'inline-block text-sm',
+              this.isAccordion() ? 'grow' : 'w-full',
+              horizontalPaddingBottom,
+              this.calculatePaddingX()
+            )}
+          ></slot>`
+        : null;
+
+    const childrenSlot = slots['children'] && !this.horizontal ? html`<slot name="children"></slot>` : null;
```

Comment:
In general we try to have the HTML as combined as possible to get a better feeling for the final outcome. Maybe you could give that a try, espcially as some slots are now going to be removed. :)